tests/cupy_tests/linalg_tests/test_decomposition.py

```diff
@@ -185,7 +188,7 @@ def check_singular(self, shape, xp, dtype):
         return result
 
     def check_rank2(self, array):
-        with self.assertRaises(numpy.linalg.LinAlgError):
+        with pytest.raises(numpy.linalg.LinAlgError):
```

Comment:
This line fixes the test to pytest style in the class.  Could you add `import pytest`? 
```diff
--- a/tests/cupy_tests/linalg_tests/test_decomposition.py
+++ b/tests/cupy_tests/linalg_tests/test_decomposition.py
@@ -1,6 +1,7 @@
 import unittest

 import numpy
+import pytest
 import six

 import cupy
```

Reply:
Thanks, I've fixed!